spec/models/user_favorite_organization_spec.rb

```diff
@@ -0,0 +1,6 @@
+require "rails_helper"
+
+RSpec.describe UserFavoriteOrganization, type: :model do
+  it { should belong_to(:user) }
+  it { should belong_to(:organization) }
```

Comment:
@arincon9 can you add a test for the validation that ensures that a user cannot favorite the same organization twice?
`  validates :user_id, uniqueness: { scope: :organization_id }`

Reply:
Oops, must have missed this. Thanks for catching it. The spec has been added.